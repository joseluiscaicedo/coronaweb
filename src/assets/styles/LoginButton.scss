@import "./Vars.scss";
Button{
  background: none;
  border:none;
  color: $third-color;
}

Button:hover{
cursor: pointer;
color:$main-color;
transition: 2s;
}