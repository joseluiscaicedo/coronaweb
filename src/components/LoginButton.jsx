/* eslint-disable import/prefer-default-export */
import React from 'react';
import { useAuth0 } from '@auth0/auth0-react';

export const LoginButton = () => {
  const { loginWithRedirect } = useAuth0();

  return <button type='button' onClick={() => loginWithRedirect}>Login</button>;
};

