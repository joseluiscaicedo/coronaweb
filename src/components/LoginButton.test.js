import React from 'react';
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import LoginButton from './LoginButton';


describe('validate LoginButton Component', () =>{
  test('renders content', () => {
    const component = render(<LoginButton />)

    expect(component.length)>0;

  });
  test('Generates Snapshot for LoginButton Component', () => {
    const { container } = render(
      <LoginButton />
    );
    expect(container.firstChild).toMatchSnapshot();
  });

})