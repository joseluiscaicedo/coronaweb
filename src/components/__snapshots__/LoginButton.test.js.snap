// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate LoginButton Component Generates Snapshot for LoginButton Component 1`] = `
<button
  class="ButtonLogin"
  id="ButtonLogin"
  type="button"
>
  Login
</button>
`;
